parquet-avro/src/main/java/org/apache/parquet/avro/AvroSchemaConverter.java

```diff
@@ -538,7 +544,7 @@ public Optional<LogicalType> visit(
             LogicalTypeAnnotation.TimeUnit unit = timestampLogicalType.getUnit();
             boolean isAdjustedToUTC = timestampLogicalType.isAdjustedToUTC();
 
-            if (isAdjustedToUTC) {
+            if (isAdjustedToUTC || !avroVersionSupportsLocalTimestampTypes()) {
```

Comment:
If local timestamps aren't supported, isn't it more safe to default to `empty()`? Otherwise, we're mixing up types.

Reply:
sorry for the confusion! changes to both methods should be needed; I've added test cases to `TestAvroSchemaConverter` that demonstrate the issue and fail without my changes to `AvroSchemaConverter` applied. Basically, we don't want to try to assign a logical type that doesn't exist in the user's Avro version.

Reply:
No problem, thanks for adding the test 👍